processor/src/operations/io_ops.rs

```diff
@@ -181,10 +181,10 @@ where
         Ok(())
     }
 
-    /// Moves 8 elements from the head of the advice tape to memory via the stack.
+    /// Moves 8 elements from the head of the advice stack to memory via the stack.
     ///
     /// The operation works as follows:
-    /// - Two words are read from the head of the advice tape.
+    /// - Two words are read from the head of the advice stack.
```

Comment:
Let's change this to: "Two words are popped from the top of the advice stack."

Reply:
Done!